packages/benchmarks/src/models.ts

```diff
@@ -110,66 +112,98 @@ export const models: ModelConfig[] = [
+    deployment: "us.anthropic.claude-3-sonnet-20240229-v1:0",
     developer: "Anthropic",
     maxConcurrency: 1,
-    provider: "radiant",
+    provider: "aws_bedrock",
     authorized: true,
   },
   {
     label: "claude-3-haiku",
-    deployment: "anthropic.claude-3-haiku-20240307-v1:0",
+    deployment: "us.anthropic.claude-3-haiku-20240307-v1:0",
     developer: "Anthropic",
     maxConcurrency: 3,
-    provider: "radiant",
+    provider: "aws_bedrock",
     authorized: true,
   },
   {
     label: "claude-35-sonnet",
-    deployment: "anthropic.claude-3-5-sonnet-20240620-v1:0",
+    deployment: "us.anthropic.claude-3-5-sonnet-20240620-v1:0",
     developer: "Anthropic",
     maxConcurrency: 1,
-    provider: "radiant",
+    provider: "aws_bedrock",
+    authorized: true,
+  },
+  {
+    label: "claude-35-sonnet-v2",
+    deployment: "us.anthropic.claude-3-5-sonnet-20241022-v2:0",
+    developer: "Anthropic",
+    maxConcurrency: 1,
+    provider: "aws_bedrock",
+    authorized: true,
+  },
+  {
+    label: "claude-35-haiku",
+    deployment: "us.anthropic.claude-3-5-haiku-20241022-v1:0",
+    developer: "Anthropic",
+    maxConcurrency: 1,
+    provider: "aws_bedrock",
     authorized: true,
   },
   {
     label: "llama-3-70b",
     deployment: "meta.llama3-70b-instruct-v1:0",
     developer: "Meta",
     maxConcurrency: 1,
-    provider: "radiant",
+    provider: "aws_bedrock",
     authorized: true,
   },
   {
     label: "mistral-large-2",
     deployment: "mistral.mistral-large-2402-v1:0",
     developer: "Mistral",
     maxConcurrency: 1,
-    provider: "radiant",
+    provider: "aws_bedrock",
     authorized: true,
   },
   {
     label: "llama-3.1-70b",
-    deployment: "accounts/fireworks/models/llama-v3p1-70b-instruct",
+    deployment: "us.meta.llama3-1-70b-instruct-v1:0",
     developer: "Meta",
     maxConcurrency: 1,
-    provider: "braintrust",
-    metadata: {
-      modelHost: "Fireworks",
-    },
-    authorized: false,
+    provider: "aws_bedrock",
+    authorized: true,
   },
   {
     label: "llama-3.2-90b",
-    deployment: "accounts/fireworks/models/llama-v3p2-90b-vision-instruct",
+    deployment: "us.meta.llama3-2-90b-instruct-v1:0",
     developer: "Meta",
     maxConcurrency: 1,
-    provider: "braintrust",
-    metadata: {
-      modelHost: "Fireworks",
-    },
-    authorized: false,
+    provider: "aws_bedrock",
+    authorized: true,
+  },
+  {
+    label: "nova-lite-v1:0",
+    deployment: "amazon.nova-lite-v1:0",
+    developer: "Amazon",
+    provider: "aws_bedrock",
+    authorized: true,
+  },
+  {
+    label: "nova-micro-v1:0",
+    deployment: "amazon.nova-micro-v1:0",
+    developer: "Amazon",
+    provider: "aws_bedrock",
+    authorized: true,
+  },
+  {
+    label: "nova-pro-v1:0",
+    deployment: "amazon.nova-pro-v1:0",
+    developer: "Amazon",
+    provider: "aws_bedrock",
+    authorized: true,
   },
+  // Note: not available in Bedrock (yet?)
```

Comment:
Do we need a follow up EAI ticket?

Reply:
no, i dont think so. like bedrock doesn't support it currently. 

when bedrock adds more models, we can add them. 

i dont think it's a big deal that we're missing this one specifically